source/plugin/economy/using.rst

```diff
@@ -34,18 +29,24 @@ Example: Loading the EconomyService
 
 .. code-block:: java
 
-	import org.spongepowered.api.event.Listener;
-	import org.spongepowered.api.event.service.ChangeServiceProviderEvent;
 	import org.spongepowered.api.service.economy.EconomyService;
+	import org.spongepowered.api.Sponge;
 
-	private EconomyService economyService;
-
-	@Listener
-	public void onChangeServiceProvider(ChangeServiceProviderEvent event) {
-		if (event.getService().equals(EconomyService.class)) {
-			economyService = (EconomyService) event.getNewProviderRegistration().getProvider();
-		}
+	Optional<EconomyService> serviceOpt = Sponge.getServiceManager().provide(EconomyService.class);
+	if (!serviceOpt.isPresent()) {
+	    // handle silly users who don't install economy plugins on their servers
```

Comment:
Why make the assumption that there must be an economy implementation?

Something like "Handle the case where an economy service is not available" would be better IMO